### ISSUE TITLE
Passwords including '@' are not supported

### ISSUE THREAD

**Issue by RobMin** [ysde/grafana-backup-tool#170]
Doing a backup using the admin account and password workflow breaks whenever 'ADMIN_PASSWORD' contains '@'.

Repro:

On clean VM with `google/cloud-sdk:367.0.0` base image do
```
pip install grafana-backup==1.2.2
export GRAFANA_URL=https://my.grafana.url
export GCS_BUCKET_NAME=my-grafana-backup
export ADMIN_ACCOUNT=admin
export ADMIN_PASSWORD='some@pass!word'
grafana-backup save
```
throws after successful `/api/health` check
```
[Pre-Check] grafana auth check: https://my.grafana.url/api/auth/keys
[DEBUG] resp status: 401
[DEBUG] resp body: {'message': 'Unauthorized'}
Traceback (most recent call last):
  File "/usr/local/bin/grafana-backup", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/dist-packages/grafana_backup/cli.py", line 52, in main
    save(args, settings)
  File "/usr/local/lib/python3.7/dist-packages/grafana_backup/save.py", line 32, in main
    (status, json_resp, dashboard_uid_support, datasource_uid_support, paging_support) = api_checks(settings)
ValueError: not enough values to unpack (expected 5, got 4)
```
The same issue appears when using `export ADMIN_PASSWORD='some%40pass!word'`

I tried using `admin1` as a password and it is working fine.

**Comment by RobMin** [comment]
Update:
I tried creating another VM and `username=admin1` `password=admin1` didn't work.
And now I'm confused is it supposed to work with only `ADMIN_ACCOUNT` and `ADMIN_PASSWORD`?

**Comment by yasinlachiny** [comment]
ADMIN_ACCOUNT and ADMIN_PASSWORD not working for me either.

**Comment by acjohnson** [comment]
@yasinlachiny and @RobMin can you please try creating an admin token instead? According to the readme I'm pretty sure the only things that can be backed up using `ADMIN_ACCOUNT` and `ADMIN_PASSWORD` is Organizations and Users.

**Comment by RobMin** [comment]
In my case, grafana is being cleaned once in a while, and creating a token manually every time is not practical.
So I wrote a script to generate a token with a short lifespan which is then passed to `grafana-backup save` with an env variable.
And I believe a feature like this should be included in the tool.

Dashboards can also be backed up using only `ADMIN_ACCOUNT` and `ADMIN_PASSWORD`(I tried to hit the API using `curl` with -u 'admin:admin') even if it is not included in the docs.

**Comment by yasinlachiny** [comment]
I can create backup by API Token and everything works fine but I want to create backup from user too.
I used grafana cloud. maybe they do not support user and password

**Comment by acjohnson** [comment]
Just to reiterate. `grafana-backup-tool` is unable to backup all component types with only the username and password and you MUST provide an admin token (`GRAFANA_TOKEN` env var). Also I just want to point out the environment variables you are using here are not correct and should be using `GRAFANA_ADMIN_ACCOUNT` and `GRAFANA_ADMIN_PASSWORD`

**Comment by yasinlachiny** [comment]
I used GRAFANA_ADMIN_ACCOUNT and GRAFANA_ADMIN_PASSWORD but I got permission denied.
I'm pretty sure that my username and password is correct

**Comment by acjohnson** [comment]
@yasinlachiny are you providing a token as well? You have to provide a token if you want to be able to backup dashboards as well as users and organizations.

Regards

**Comment by yasinlachiny** [comment]
Yes I used token as well. I can create backup from dashboards but I can't create backup from users. I talked with grafana supports they say that grafana cloud does not support username and password.
They said:

Unfortunately the Basic Auth method on this is not available on hosted Grafana Cloud instances. So the credentials you supplied probably were correct, it's just not allowed via the API itself.
